aiohttp/wsgi.py

```diff
@@ -105,7 +105,7 @@ def create_wsgi_environ(self, message, payload):
         if script_name:
             path_info = path_info.split(script_name, 1)[-1]
 
-        environ['PATH_INFO'] = path_info
+        environ['PATH_INFO'] = unquote(path_info).encode().decode('latin-1')
```

Comment:
Shouldn't here be `utf-8` instead of `latin-1`? Since you turns all multilang chars here into garbage.